source/MapPanel.cpp

```diff
@@ -218,13 +218,14 @@ void MapPanel::Draw()
 
 	// Draw the "visible range" circle around your current location.
 	Color dimColor(.1f, 0.f);
-	RingShader::Draw(Zoom() * (playerSystem.Position() + center),
-		(System::DEFAULT_NEIGHBOR_DISTANCE + .5) * Zoom(), (System::DEFAULT_NEIGHBOR_DISTANCE - .5) * Zoom(), dimColor);
+	RingShader::Draw(Zoom() * (playerSystem.Position() + center), System::DEFAULT_NEIGHBOR_DISTANCE * Zoom(), 2.0f, 1.0f,
+		dimColor, 25.0, 10.0);
```

Comment:
```suggestion
		dimColor);
```

The dashing is so low you might as well drop it completely.